motionplan/tpSpaceRRT.go

```diff
@@ -743,11 +740,7 @@ func (mp *tpSpaceRRTMotionPlanner) smoothPath(ctx context.Context, path []node)
 			maxCost = wp.Cost()
 		}
 	}
-	newFrame, err := tpspace.NewPTGFrameFromPTGFrame(mp.frame, maxCost*mp.algOpts.smoothScaleFactor, 0)
```

Comment:
Looks like we may be able to remove this function now that it's not being used here as this was the only instance of that.

Reply:
I'll add a TODO referencing RSDK-5104, and determine what to do with the function in that ticket.